ggml/src/ggml-cuda/ggml-cuda.cu

```diff
@@ -242,6 +242,7 @@ static ggml_cuda_device_info ggml_cuda_init() {
 
         info.devices[id].nsm   = prop.multiProcessorCount;
         info.devices[id].smpb  = prop.sharedMemPerBlock;
+        info.devices[id].warp_size = prop.warpSize;
```

Comment:
```suggestion
        info.devices[id].nsm       = prop.multiProcessorCount;
        info.devices[id].smpb      = prop.sharedMemPerBlock;
        info.devices[id].warp_size = prop.warpSize;
```

Reply:
Done, but i hate this, this kind of alignment makes merges harder, blame less useful and diffs harder to read.